README.md

```diff
@@ -290,11 +290,14 @@ $ lerna publish --canary --yes
 When run with this flag, `publish` will skip all confirmation prompts.
 Useful in [Continuous integration (CI)](https://en.wikipedia.org/wiki/Continuous_integration) to automatically answer the publish confirmation prompt.
 
-#### --repo-version
+#### --update-version [repo, package], -u
 
 ```sh
-$ lerna publish --repo-version 1.0.1
-# applies version and skips `Select a new version for...` prompt
+$ lerna publish --update-version "repo x.x.x"
```

Comment:
My thought process was to explicitly declare the repo command. It can be changed though. We can simply say if the flag for `--update-version` isn't `package` then test it's a valid semver and move on with `repo` logic